README_FA.md

```diff
@@ -0,0 +1,22 @@
+<div dir="rtl">
+
+# پیوستن به تیم ما
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-7-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![README.md](https://img.shields.io/badge/English-up-brightgreen)](README.md)
+[![README_JO.md](https://img.shields.io/badge/Arabic-up-brightgreen)](README_JO.md)
+[![README_PH.md](https://img.shields.io/badge/Filipino-up-brightgreen)](README_PH.md)
+[![README_RU.md](https://img.shields.io/badge/Russian-up-brightgreen)](README_RU.md)
+[![README_FA.md](https://img.shields.io/badge/Farsi-up-brightgreen)](README_FA.md)
```

Comment:
Also the language buttons should be ordered alphabetically by language name.

So Farsi should come before Filipino

Reply:
Good, But button list should be updated in all of other READMEs too.

Reply:
Would you like to add the language buttons to all the README's ?

Would be a great help to me 👍